src/controllers/movies/movies.js

```diff
@@ -31,11 +31,16 @@ export default function (view, params, tabContent, options) {
     }
 
     function shuffle() {
-        ApiClient.getItem(
-            ApiClient.getCurrentUserId(),
-            params.topParentId
-        ).then((item) => {
-            playbackManager.shuffle(item);
+        isLoading = true;
+        loading.show();
+        const newQuery = { ...query, SortBy: 'Random', StartIndex: 0, Limit: 300 };
+        return ApiClient.getItems(ApiClient.getCurrentUserId(), newQuery).then(({ Items }) => {
```

Comment:
We should probably use the `getItems` utility function: https://github.com/jellyfin/jellyfin-web/blob/master/src/utils/jellyfin-apiclient/getItems.ts#L67

Reply:
It is mostly for long `Ids`. Maybe only for consistency. 🤷‍♂️

Reply:
Ah I missed the limit being set so this is fine. 👍👍